getting_started/setup_vm/run.sh

```diff
@@ -10,5 +10,5 @@ set -ex
 sudo apt-get update
 sudo apt install software-properties-common
 sudo add-apt-repository -y --update ppa:ansible/ansible
-sudo apt install ansible-base -y
+sudo apt install ansible -y
```

Comment:
Can you also remove the comment on lines 7-9?

Reply:
Edited rather than removed, since we still need the ppa afaict.